server/index.js

```diff
@@ -0,0 +1,11 @@
+var express = require('express');
+/* We are using a single instance of Express to handle incoming requests */
+var app = express();
+
+/* Dummy router example */
+app.get('/', (req, res) => {
+    res.send({ hi: 'there' });
+});
+
+/* Instruct Express to listen to this port */
+app.listen(5000);
```

Comment:
See comments.

We basically make an instance of Express and tell it to listen to port 5000. We can make requests to this server via this dummy route on the browser. 

You can test this by running "node index.js" in the terminal and go to localhost:5000/ to see the dummy data!

Reply:
Also don't mind anything below this. I'm just moving everything inside a "server" folder